lib/util/hud_utility_2024.rb

```diff
@@ -10,24 +10,19 @@ module HudUtility2024
 
   module_function
 
-  # def race(field, reverse = false, multi_racial: false)
-  #   map = races(multi_racial: multi_racial)
-  #   _translate map, field, reverse
-  # end
-
-  # # NOTE: HUD, in the APR specifies these by order ID, as noted in the comments below
-  # def races(multi_racial: false)
-  #   race_list = {
-  #     'AmIndAKNative' => 'American Indian, Alaska Native, or Indigenous', # 1
-  #     'Asian' => 'Asian or Asian American', # 2
-  #     'BlackAfAmerican' => 'Black, African American, or African', # 3
-  #     'NativeHIPacific' => 'Native Hawaiian or Pacific Islander', # 4
-  #     'White' => 'White', # 5
-  #     'RaceNone' => 'Doesn\'t Know, refused, or not collected', # 6 (can be 99, 8, 9, null only if all other race fields are 99 or 0)
-  #   }
-  #   race_list['MultiRacial'] = 'Multi-Racial' if multi_racial
-  #   race_list
-  # end
+  def race(field, reverse = false, multi_racial: false)
+    map = multi_racial ? races_with_multi_racial : races
+    _translate map, field, reverse
+  end
+
+  # NOTE: api differs from HudUtility 2022 which was `races(multi_racial: false)`
+  # NOTE: HUD, in the APR specifies these by order ID, as noted in the comments below
```

Comment:
@gigxz this comment seems to have been disconnected from the source data.
Also, is there any reason we couldn't make a helper method `races` that had the same API as the previous version?  It's probably fine update all instances, but I'm guessing that's quite a few.

Reply:
Yeah there was no good reason, I updated it to match the old api for `races`.

I don't know why that APR comment was there, it was just referncing some commented-out numbers that were next to each race. I added a new lookup method for the race IDs since we didnt seem to have one.
```
  def race_id_to_field_name
    # Integer values from HUD Data Dictionary
    {
      1 => :AmIndAKNative,
      2 => :Asian,
      3 => :BlackAfAmerican,
      4 => :NativeHIPacific,
      5 => :White,
      6 => :HispanicLatinaeo,
      7 => :MidEastNAfrican,
      8 => :GenderNone,
      9 => :GenderNone,
      99 => :GenderNone,
    }.freeze
```

Reply:
Oh I see